packages/components/Molecules/Form/index.vue

```diff
@@ -326,6 +338,14 @@ $rpl-form-input-search-icon: url("data:image/svg+xml,%3Csvg width='16' height='1
   &--full-width {
     max-width: 100%;
   }
+
+  &__sr-only {
+    @include rpl_visually_hidden;
+    ul {
+      // Avoid screenreader reading out "bullet".
```

Comment:
@dylankelly I'm not really sure whether I understand. Can you explain it a bit more? I'm just using the standard `rpl_visually_hidden` mixin to visually hide the messages. You mean using the `rpl-visually-hidden` class instead? I wrapped it in the custom `_sr-only` just because it needed the extra UL treatment. I guess it could be reshuffled if that's what you mean.

Reply:
Yeah maybe i didnt think that through first. What I was probably intending was a list reset mixin that included this.